src/lib/resolvers/emoji.ts

```diff
@@ -0,0 +1,30 @@
+import { EmojiRegex, TwemojiRegex } from '@sapphire/discord-utilities';
+import { Util } from 'discord.js';
+import { Identifiers } from '../errors/Identifiers';
+import { err, ok } from '../parsers/Result';
+
+export function resolveEmoji(parameter: string) {
+	const twemoji = TwemojiRegex.exec(parameter)?.[0] ?? null;
+
+	if (twemoji) {
+		return ok(twemoji);
+	}
+
+	const emojiId = EmojiRegex.exec(parameter)?.groups?.id;
+
+	if (emojiId) {
+		const resolved = Util.parseEmoji(emojiId) as EmojiObject;
+
+		if (resolved) {
+			return ok(resolved);
+		}
+	}
```

Comment:
This is always going to return `false` for `animated` as per how `parseEmoji` works: https://github.com/discordjs/discord.js/blob/12ffa069aa8b247e945fef16a543f41c2c391bf1/packages/discord.js/src/util/Util.js#L284-L298

Instead pass the full parameter into the method. 

![image](https://user-images.githubusercontent.com/4019718/148848065-2079efad-d519-4b61-9b06-0d4ada0b9d5f.png)